pkg/generators/models/testdata/cases/enum/api.yaml

```
@@ -0,0 +1,12 @@
+openapi: 3.0.0
+info:
+  version: 0.1.0
+  title: Test
+
+components:
+  schemas:
+    Foo:
```

Let's make this easier to read 
```suggestion
    Name:
```